client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -227,6 +226,10 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 
 	const handleBackClick = () => {
 		if ( isPreviewingDesign ) {
+			recordTracksEvent( 'calypso_signup_design_preview_exit', {
+				...getEventPropsByDesign( selectedDesign as Design ),
```

Comment:
A small thing, `getEventPropsByDesign` returns an object we clone here with `...`, is there a reason for it? We could avoid the clone.

```
recordTracksEvent( 'calypso_signup_design_preview_exit', getEventPropsByDesign( selectedDesign as Design ) );
```

Reply:
Good point! I'll update